debian/postinst

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+flatpak remote-add --if-not-exists --system appcenter https://flatpak.elementary.io/repo.flatpakrepo
+flatpak install --system --noninteractive appcenter org.gnome.Epiphany//stable || true
+
+flatpak install --system --noninteractive freedesktop org.freedesktop.Platform.GL.default//21.08
```

Comment:
We need to make sure to also add this remote like line 5 and not assume it exists